packages/xo-server-perf-alert/package.json

```diff
@@ -17,6 +17,7 @@
   },
   "dependencies": {
     "@xen-orchestra/cron": "^1.0.6",
+    "@xen-orchestra/log": "^0.4.0",
```

Comment:
Let's be careful in the future:

- you added an old version of `@xen-orchestra/log`, current one is v0.5.0
- you did not run `yarn` after adding a new dependency which made `yarn.lock` inconsistent with the repo.

Reply:
No problem :slightly_smiling_face: